go/host/l1/statemachine.go

```diff
@@ -160,7 +166,7 @@ func (c *crossChainStateMachine) Synchronize() error {
 	forkUID, _, _, err := c.publisher.GetBundleRangeFromManagementContract(big.NewInt(0).SetUint64(c.latestRollup.Number), c.latestRollup.ForkUID)
 	if err != nil {
 		if errors.Is(err, errutil.ErrNoNextRollup) {
-			c.logger.Debug("No new rollup or fork found")
+			c.logger.Info("No new rollup or fork found")
```

Comment:
Looks like this happens every 6 secs unless there's been a rollup? So I guess adds 600 logs per hour (kwik maffs) per node...

But we're not on datadog anymore and we need to understand this shiz so maybe small price to pay! Was just wondering if there was a way to infer that no rollup was found, e.g. because we log when we do find one.